db/migrate/20250105113940_create_contests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateContests < ActiveRecord::Migration[7.0]
+  def change
+    create_table :contests do |t|
+      t.datetime :deadline
+      t.integer :status, :integer
```

Comment:
```suggestion
      t.integer :status
```

Do we need the `integer` column on contests table? Seems like a mistake. Could you please remove this and rerun the migration

Reply:
This is accidental, will remove this.